src/appshell/view/navigableappmenumodel.cpp

```diff
@@ -277,11 +277,13 @@ bool NavigableAppMenuModel::processEventForAppMenu(QEvent* event)
                                   && isSingleSymbol
                                   && isNavigationStarted;
     bool isNavigationWithAlt = (modifiers & Qt::AltModifier)
+                               && !(modifiers & Qt::ControlModifier)
                                && !(modifiers & Qt::ShiftModifier)
```

Comment:
How about just 
```suggestion
    bool isNavigationWithAlt = (modifiers == Qt::AltModifier)
```
?

Reply:
Sure, more modifier-proof! :) Thanks.